packages/deployer/test/sample-project/contracts/libraries/Address.sol

```diff
@@ -0,0 +1,36 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+/**
+ * @dev Collection of functions related to the address type
+ */
+library Address {
```

Comment:
I would remove this for now and, if you wanna use it in the UpgradeModule, do this as an internal function.